ui/pages/confirmations/selectors/confirm.ts

```diff
@@ -43,3 +48,23 @@ export const confirmSelector = (state: ConfirmMetamaskState) => state.confirm;
 
 export const currentConfirmationSelector = (state: ConfirmMetamaskState) =>
   state.confirm.currentConfirmation;
+
+export const currentSignatureRequestSecurityResponseSelector = (
```

Comment:
Doesn't need addressing now but we have lots of competing standards for selector names such as `XSelector`, `getX`, or `selectX`.

My understanding was the latter is best as it's the most explicit, makes auto-complete easy, and is recommended by Redux.